framework/src/node/node.ts

```diff
@@ -172,14 +172,38 @@ export class Node {
 		}
 
 		if (customModule.id < MINIMUM_MODULE_ID) {
-			throw new Error(`Custom module must have id greater than ${MINIMUM_MODULE_ID}`);
+			throw new Error(`Custom module must have id greater than ${MINIMUM_MODULE_ID}.`);
 		}
 		if (customModule.accountSchema) {
 			this._registeredAccountSchemas[customModule.name] = {
 				...customModule.accountSchema,
 				fieldNumber: customModule.id,
 			};
 		}
+
+		if (customModule.transactionAssets.length > 0) {
```

Comment:
I think this `if` is not needed, since when `length === 0` the `for..of` will not execute